app/navigation/NavigatorForCorrectReturn.scala

```diff
@@ -89,8 +88,10 @@ class NavigatorForCorrectReturn @Inject()() extends Navigator {
   }
 
   override val normalRoutes: Page => UserAnswers => Call = {
+
     case RemovePackagingSiteConfirmPage => _ => defaultCall
     case SmallProducerDetailsPage => _ => defaultCall
+    case PackagingSiteDetailsPage => _ => defaultCall
```

Comment:
We might need to think about the navigation as three different variation journeys end up on this call and they may have different navigation requirements but we'll cross that bridge when we come to it.